routes/index.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const router = express.Router();
+const path = require( 'path' );
+const { db, getAllBooks, getBook, addBook, updateBook, removeBooks } = require('../database/dbBookStore')
+
+
+router.get( '/', ( req, res, next ) => {
+  getAllBooks()
+    .then( books => {
+    res.render( 'index', { books, title: 'Bookstore App using React' } )
+  })
```

Comment:
Add a .catch(next) to all of these functions. That way if an error is thrown it will get passed down until it hits an error handler. Your error handlers are in the lower half of app.js